wallet/src/de/schildbach/wallet/Constants.java

```diff
@@ -268,7 +268,7 @@ public void log(final String message) {
     private static final Logger log = LoggerFactory.getLogger(Constants.class);
 
     //Dash Specific
-    public static long EARLIEST_HD_SEED_CREATION_TIME = 1427610960l;
+    public static long EARLIEST_HD_SEED_CREATION_TIME = 1585699200L;    //TODO change me back to 1427610960L when testing is done
```

Comment:
This needs to be reverted before merging.

Reply:
Done